src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BALL_NUMBER = 100;
+    private Random random = new Random();
+    private ColorSupplier colSup = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColour(colSup.getRandomColor().toString());

```

Comment:
```suggestion
        ball.setColour(colSup.getRandomColor().name());
```
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)
Use name() for getting String representation of enum constants

Reply:
+